### ISSUE TITLE
Connection of TXD and RXD of the u-blox

### ISSUE THREAD

**Issue by ramapongithub** [bazjo/RS41_Hardware#3]
Hi, 
there seems to be a small error in the schematic of the GPS-part.
According to the scan the capacitors C317 and C318 are directly connected to pin 21 resp. pin 24 of the u-blox on one side and GND on the other side.
![rear2](https://user-images.githubusercontent.com/16646472/65500516-b64b0380-debf-11e9-856e-f0193badfebf.jpg)
 
But the schematic shows them in series with the resistors R306 resp. R307, between resistor and GND.

![gps_sch](https://user-images.githubusercontent.com/16646472/65500727-03c77080-dec0-11e9-9afc-64003f5de032.png)

**Comment by bazjo** [comment]
Totally right, must have gone through. Will fix when I'm at the schematic next time. Shouldn't be a big deal when interfacing with the sonde though

**Comment by ramapongithub** [comment]
Hold your horses and take your time. There may be one or two other issues around there. I will dig a little deeper and report.

**Comment by ramapongithub** [comment]
Found what bothered me.
Between this two points is continuity,
![rear2_2](https://user-images.githubusercontent.com/16646472/65666934-06040900-e03f-11e9-9d28-100d3bde7f2d.jpg)
wich means that +3V_GPS is connected to the point between R301 and R302. This provides power to the oscillator and VDD_3V. I measured 0 Ohm at R301 and R302 , so probably acting as fuses.
Schematic should then look something like this:
![gps_sch1](https://user-images.githubusercontent.com/16646472/65667740-9d1d9080-e040-11e9-8600-1bd12065668d.jpg)
